SDVModTest/UIElements/ShowItemHoverInformation.cs

```diff
@@ -208,11 +208,17 @@ private void DrawAdvancedTooltip()
                     }
                 }
 
-                int largestTextWidth = 0;
-                int stackTextWidth = (int)(Game1.smallFont.MeasureString(stackPrice.ToString()).Length());
+
+                int bundleTextWidth = 0;
+                if (!String.IsNullOrEmpty(requiredBundleName))
+                {
+	                bundleTextWidth = (int)Game1.dialogueFont.MeasureString(requiredBundleName).Length();
+	                bundleTextWidth -= 30; //Text offset from left
+                }
+				int stackTextWidth = (int)(Game1.smallFont.MeasureString(stackPrice.ToString()).Length());
                 int itemTextWidth = (int)(Game1.smallFont.MeasureString(itemPrice.ToString()).Length());
-                largestTextWidth = (stackTextWidth > itemTextWidth) ? stackTextWidth : itemTextWidth;
-                int windowWidth = Math.Max(largestTextWidth + 90, String.IsNullOrEmpty(requiredBundleName) ? 100 : 300);
+	            int largestTextWidth = Math.Max(bundleTextWidth,Math.Max(stackTextWidth, itemTextWidth));

```

Comment:
Please remove tabs and keep spacing consistent with what I'm using (4 spaces per tab)